scripts/mkstatic.py

```diff
@@ -667,11 +667,16 @@ def parse(self, make_call_tree=False):
         suite_xml = tree.getroot()
         self._name = suite_xml.get('name')
         # Validate name of suite in XML tag against filename; could be moved to common.py
-        if not (os.path.basename(self._sdf_name) == 'suite_{}.xml'.format(self._name)):
-            logging.critical("Invalid suite name {0} in suite definition file {1}.".format(
-                                                               self._name, self._sdf_name))
-            success = False
-            return success
+        if not (os.path.basename(self._sdf_name) == '{}.xml'.format(self._name)):
+            if (os.path.basename(self._sdf_name) == 'suite_{}.xml'.format(self._name)):
+                logging.debug("Parsing suite using legacy naming convention")
+                logging.debug(f"Filename {os.path.basename(self._sdf_name)}")
+                logging.debug(f"Suite name {format(self._name)}")
+            else:
+                logging.critical("Invalid suite name {0} in suite definition file {1}.".format(
```

Comment:
Is it invalid or just not found?

Reply:
This check is for inconsistency between the SDF filename and the `<suite name=>` tag which actually defines the name of the suite (`Suite._name`). So when reading in a file `suite_abcd.xml`, the only two valid values for `Suite._name` would be `suite_abcd` or `abcd`. Any other value is invalid.

If you think this error message is too unclear I could update it to be more precise, since it's really about inconsistency and not necessarily "validity".

Reply:
Sorry, I misunderstood what you were checking, I now think the message is okay. Thanks for the explanation.